ega-data-api-res/src/main/java/eu/elixir/ega/ebi/reencryptionmvc/service/internal/CacheResServiceImpl.java

```diff
@@ -223,6 +215,27 @@ private static void byte_increment_fast(byte[] data, long increment) {
             cur_pos--;
         }
     }
+    
+    private String printmemory() {
```

Comment:
I understand why you want this but this is not the place to put it, I suggest you to look at the spring-boot actuator. That information should be easily queriable from there and import into a metrics system to have a real time report of the application. Here is only gonna be fluff on the log.

Reply:
Done